src/common/config/env.validation.ts

```diff
@@ -98,78 +106,102 @@ export class EnvironmentVariables {
   @Transform(({ value }) => value || LogFormat.json)
   LOG_FORMAT: LogFormat = LogFormat.json;
 
+  @IsNotEmpty()
   @IsArray()
   @ArrayMinSize(1)
-  @Transform(({ value }) => value.split(',').map((url) => url.replace(/\/$/, '')))
+  @IsUrl({
+    require_protocol: true
+  }, {
+    each: true,
+  })
+  @Transform(({ value }) => toArrayOfUrls(value))
   PROVIDERS_URLS!: NonEmptyArray<string>;
 
-  @IsInt()
+  @IsNotEmpty()
+  @IsEnum(Network)
   @Transform(({ value }) => parseInt(value, 10))
-  CHAIN_ID!: number;
+  CHAIN_ID!: Network;
```

Comment:
I think it should be named Chain instead of Network

Reply:
Yep, I agree. `Chain` is much better. There was also a good idea to use this constant
`import { CHAINS } from '@lido-nestjs/constants';`
instead of the custom enum. If you don't mind, I'll do it this way.

Reply:
i prefer to write for kapi CHAINS enum, that will contain chains that kapi support

Reply:
Yes, agree with you.

Reply:
so it is mainnet, goerli and holesky